psig_matcher/experiments/run_experiment.py

```diff
@@ -136,17 +164,26 @@ def run_experiment(part_type: str, part_dim: int, num_samples: int, meta_pdf_ci:
 def main():
     """ This script can be run as such:
     python3 psig_matcher/experiments/run_experiment.py --part_type=CON --part_dim=5 --num_samples=100 --meta_pdf_ci=0.999 --part_pdf_ci=0.999 --confidence_bound=0.999 """
-    
+
     parser = argparse.ArgumentParser()
     parser.add_argument('--part_type', type=str, required=True)
     parser.add_argument('--part_dim', type=int, required=True)
     parser.add_argument('--num_samples', type=int, required=True)
     parser.add_argument('--meta_pdf_ci', type=float, required=True)
     parser.add_argument('--part_pdf_ci', type=float, required=True)
     parser.add_argument('--confidence_bound', type=float, required=True)
-    
+
     args = parser.parse_args()
+
+    # capture the system hyperparameters that were used when running the experiment
+    log_param("part_type", args.part_type)
+    log_param("part_dim", args.part_dim)
+    log_param("num_samples", args.num_samples)
+    log_param("meta_pdf_ci", args.meta_pdf_ci)
+    log_param("part_pdf_ci", args.part_pdf_ci)
+    log_param("confidence_bound", args.confidence_bound)
+
```

Comment:
As discussed, set experiment name here

Reply:
what is the naming standard? currently it's 0, 1, 2, ...